src/data/tournament.py

```diff
@@ -1051,11 +1051,11 @@ def _estimate_players(self, players: Iterable[Player], *, after_round: int):
 
         # NOTE(Amaras): Because EM did not take into account HPB in his code,
         # this function must be used instead of Player.points_after
-        def papi_points_after(player: Player, after_round: int) -> float:
+        def papi_points_after(player: Player, after_round_: int) -> float:
```

Comment:
```suggestion
        def papi_points_after(player: Player) -> float:
```
uses the outer scope variable already (usages to also remove)